npm-react-codemod/test/pure-render-mixin-test2.output.js

```diff
@@ -1,8 +1,9 @@
-var React = require('react/addons');
+var React = require('react');
+var shallowCompare = require('react/addons/shallowCompare');
 
 var MyComponent = React.createClass({
   shouldComponentUpdate: function(nextProps, nextState) {
-    return React.addons.shallowCompare(this, nextProps, nextState);
```

Comment:
yeah, this actually requires changing the codemod script as well to output the proper require statement.

Then in `npm-react-codemod` you can run `npm test` to check if everything still works.